python/cuspatial/cuspatial/core/binpreds/binpreds.py

```diff
@@ -154,11 +168,14 @@ def preprocess(self, lhs, rhs):
         point_indices = geom.point_indices()
         from cuspatial.core.geoseries import GeoSeries
 
-        final_rhs = GeoSeries(
-            GeoColumn._from_points_xy(xy_points._column)
-        ).points
+        final_rhs = GeoSeries(GeoColumn._from_points_xy(xy_points._column))
         return (lhs, final_rhs, point_indices)
 
+    def _should_use_quadtree(self):
```

Comment:
I thought the code for this function was self explanatory: if there are more than 31 polygons in the lhs, it has multipolygons, or the user specified allpairs.

Reply:
I think you should mention this is because the byte limited API supports only 31 polygons, and list reasons why you don't use pairwise_PiP instead. Otherwise this would look like arbitrary decisions to external contributors.

Reply:
Done.